doc/webui/README.md

```diff
@@ -0,0 +1,66 @@
+# elektra web
+
+_a web user interface (Web UI) to remotely manage multiple elektra instances_
+
+
+## Overview
+
+Elektra web consists of multiple components:
+
+ * (multiple) servers running an elektra daemon (`elektrad`)
+ * a single cluster management server to communicate with the elektra daemons (`clusterd`)
```

Comment:
I disagree, because he references the architecture within his API description. The component description in this document isn't really extensive, so I don't see a problem at all. He could still write a lot more about it in another document, that is correct.